### PR TITLE
Draft Project Portfolio Page

### PR DESCRIPTION
**Pull request by cnivedit**

### CODE REVIEW

File: docs/team/cnivedit.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+PILL is fully written in Java, and users can interact with the application using
+Command Line Interface (CLI).
+
+### Summary of Contributions
+
+- **New Feature**: Added ListCommand class
+    - What it does: The class is responsible for handling the execution of the list command ([#7](https://github.com/AY2425S1-CS2113-W14-4/tp/pull/7)).
+    - Justification: Abstraction of the list command, inheriting from the base command class.
+    - Highlights: Receives ItemMap instance and lists items.
```
Comment (yakultbottle) [inline docs/team/cnivedit.md:15]: Looks good, but you could consider making more difficult to implement features take up more space/be at the top, such that the most impressive features are easier to notice

**Review by yakultbottle — APPROVED** [review]
LGTM, approved

File: docs/team/cnivedit.md (hunk 1 above)
Comment (cnivedit) [reply docs/team/cnivedit.md]: Updated list order in cf91040

**Review by yakultbottle — APPROVED** [review]
LGTM, thanks for your hard work

**Pull request merged**